packages/deployment/app/cumulus_api.template.yml

```diff
@@ -178,6 +178,9 @@ Resources:
       Environment:
         Variables:
           stackName: {{../stackName}}
+          system_bucket: {{../parent.system_bucket}}
+          public_buckets: {{#collectBuckets ../parent.buckets "public"}}{{/collectBuckets}}
```

Comment:
So here I tried to use non-block helpers in handlebars, but couldn't get them to work.  Any ideas?
I was trying `{{{#collectBuckets ../parent.buckets "public"}}}` like the docs said, but no dice.

Reply:
Really I wanted to add these helpers to packages/api/config/distribution.yml, but was unable to get them to be interpreted in any manner.  Am I missing something fundamental about how kes works. .

something like ```  envs:
    EARTHDATA_BASE_URL: '{{parent.urs_url}}'
    EARTHDATA_CLIENT_ID: '{{EARTHDATA_CLIENT_ID}}'
    EARTHDATA_CLIENT_PASSWORD: '{{EARTHDATA_CLIENT_PASSWORD}}'
    public_buckets: '{{{#collectBuckets buckets "public"}}}'
    protected_buckets: '{{{#collectBuckets buckets "protected"}}}'
```  in this location https://github.com/nasa/cumulus/blob/master/packages/api/config/distribution.yml#L9-L12